src/components/Button/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react'
+import { checkButton } from '../../utils/checkButton'
+import style from './Button.module.scss'
+
+interface IButtonProps {
+  children?: React.ReactNode
+  handleOnClick: () => void
+  buttonType: string
```

Comment:
Add type for button types

For example 
`type TButtonStyles = "green-btn" | "card-btn"`

And add this type to interface

buttonType: TButtonStyles;